cmd/cmd.go

```diff
@@ -614,11 +617,56 @@ func versionString(manager *Manager) string {
 	if GitHash != "" {
 		suffix = fmt.Sprintf(" hash %s\n", GitHash)
 	}
-	return fmt.Sprintf("%s version %s.%s", manager.name, manager.version, suffix)
+	return fmt.Sprintf("Client version: %s.%s", manager.version, suffix)
+}
+
+func apiVersionString(client *Client) (string, error) {
+	if client == nil {
+		return "", fmt.Errorf("Null Client Exception")
+	}
+
+	url, err := GetURL("/info")
+	if err != nil {
+		return "", err
+	}
+
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", err
+	}
+
+	serverVersionPrefix := "Server version: "
+	resp, err := client.Do(req)
+	if err != nil {
+		if isUnauthorized(err) {
+			return fmt.Sprintf("%s Can't retrieve server version, user %v", serverVersionPrefix, err), nil
```

Comment:
Instead handling only the unauthorized case, what about turning this error message more generic to fit w/ any error? I'm thinking something like: `fmt.Sprintf("Unable to retrieve server version: %v", err)`.

I also think it's better suppress the `serverVersionPrefix` on error cases... I had suggested just the opposite before, my bad.